clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2418,6 +2418,15 @@ used on the same loop in conjunction with disable_loop_pipelining.
   }];
 }
 
+def SYCLIntelFPGANofusionAttrDocs : Documentation {
+  let Category = DocCatVariable;
+  let Heading = "intel::nofusion";
+  let Content = [{
+This attribute applies to a loop. Indicates that the annotated
+loop should not be fused with any adjacent loop.
```

Comment:
Should we provide some code example here?

Reply:
It seems like all SYCL loop attributes are documented without any examples. To be consistent, i did not add any code examples here.

Reply:
Okay, I think we can do it as a separate task for all SYCL loop attributes.

Reply:
Thanks Mariya. i have created and assigned to me to do this task separately: 
 https://github.com/intel/llvm/issues/2727